jans-cedarling/cedarling/src/log/memory_logger.rs

```diff
@@ -9,12 +9,12 @@ use super::interface::{LogStorage, LogWriter};
 use crate::models::log_config::MemoryLogConfig;
 use crate::models::log_entry::LogEntry;
 use sparkv::{Config as ConfigSparKV, SparKV};
-use std::{cell::RefCell, time::Duration};
+use std::{sync::Mutex, time::Duration};
 
 /// A logger that store logs in-memory.
 pub(crate) struct MemoryLogger {
     // Using RefCell to enable interior mutability
```

Comment:
Since you're now using a `Mutex`, please remove this.

Reply:
Thanks, done